include/alpaka/core/UniqueId.hpp

```diff
@@ -32,7 +32,7 @@ namespace alpaka
         namespace detail
         {
             //#############################################################################
-            // Based on the code from Filip Ros�en at http://b.atch.se/posts/constexpr-counter/
+            // Based on the code from Filip Ros�en at http://b.atch.se/posts/constexpr-counter/
```

Comment:
ohh yes. We should encode all files with plain ascii and not with UTF8. There are some compiler were we get problems with UTF8:-(